udfs/tests/dataform_testing_framework/deploy_and_run_tests.sh

```diff
@@ -240,12 +240,17 @@ main() {
     local udf_dirs
     # Get the list of directory names which contain UDFs
     udf_dirs=$(sed 's/:.*//g' <../../dir_to_dataset_map.yaml)
+    region_suffix=$(sed -rn "s/${BQ_LOCATION}: (.*)/\1/p" <../../region_to_dataset_suffix_map.yaml)
 
     for udf_dir in ${udf_dirs}; do
       # Get the short-hand version of the dataset_id
       # which is mapped in dir_to_dataset_map.yaml
       local dataset_id
       dataset_id=$(sed -rn "s/${udf_dir}: (.*)/\1/p" <../../dir_to_dataset_map.yaml)
+      # Region suffixes are used to deploy UDFs globally to bqutil without naming conflicts
+      if [[ "${PROJECT_ID}" = "bqutil" && "${BQ_LOCATION}" != "US" ]]; then
```

Comment:
```suggestion
      if [[ "${PROJECT_ID}" = "bqutil" && "${BQ_LOCATION^^}" != "US" ]]; then
```

Make location comparison case insensitive using [bash param expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html#:~:text=The%20%E2%80%98%5E%E2%80%99%20operator%20converts%20lowercase%20letters%20matching%20pattern%20to%20uppercase%3B%20the%20%E2%80%98%2C%E2%80%99%20operator%20converts%20matching%20uppercase%20letters%20to%20lowercase.%20The%20%E2%80%98%5E%5E%E2%80%99%20and%20%E2%80%98%2C%2C%E2%80%99%20expansions%20convert%20each%20matched%20character%20in%20the%20expanded%20value%3B)